ios/keyman/Keyman/Keyman/MainViewController.swift

```diff
@@ -110,14 +110,14 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
     // We have to gerry-rig this so that the framework-based SettingsViewController
     // can launch the app-based DocumentViewController.
     if #available(iOS 11.0, *) {
-      Manager.shared.fileBrowserLauncher = { _ in
+      Manager.shared.fileBrowserLauncher = { navVC in
```

Comment:
Declaration: https://github.com/keymanapp/keyman/blob/b205d1efff970babb0eb5aea5ef89b6a69f97a77/ios/engine/KMEI/KeymanEngine/Classes/Manager.swift#L47

Turns out the parameter was declared for a reason.  Who knew?